### PR TITLE
Core 18 security takeover and landing page

### PR DESCRIPTION
**Pull request by solazio**
## Done

Add Core 18 security takeover and landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/) and [/engage/core18-security](http://0.0.0.0:8001//engage/core18-security)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to home page and refresh the page until you see the Core 18 takeover. After you check it, click on "Watch the webinar" button and check the landing page
- Make sure it complies with the [brief](https://docs.google.com/document/d/1i_Zo_IdVsGJlMtDKXQh77H_LtbNyS4_vie6vBTHnxbw/edit#)
- Make sure it complies with the [design](https://github.com/canonical-web-and-design/web-squad/issues/1170) and [more design](https://github.com/canonical-web-and-design/web-squad/issues/1178)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1178

### CODE REVIEW

**Comment by anasereijo** [comment]
I think it all looks good. 👍 

Is it possible to left start left aligning the video iframe?
Screenshot 2019-05-30 at 09 41 41

**Comment by anasereijo** [comment]
@WendyS1000 @caldav can you please check if everything is ok to you?

File: templates/takeovers/_core18-security_takeover.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <h4 class="p-heading--four">
+        Why Ubuntu Core is the most secure choice for IoT.
+      </h4>
+      <ul class="p-inline-list u-hide--small">
+        <li class="p-inline-list__item">
+          <a href="/engage/core18-security?utm_source=Takeover&utm_medium=Takeover&utm_campaign=CY19_IOT_UbuntuCore_WBN_Core18Security" class="p-button--positive"><span>Watch the webinar</span></a>
+        </li>
+      </ul>
+    </div>
+      <div class="col-6">
```
Comment (sowasred2012) [inline templates/takeovers/_core18-security_takeover.html]: This block is indented one level too many

**Comment by WendyS1000** [comment]
> @WendyS1000 @caldav can you please check if everything is ok to you?

@anasereijo looks fine to me.

**Comment by solazio** [comment]
@anasereijo I cannot change this as it is part of the script I get in the brief. Maybe you can talk to them and ask them to remove left margin.

**Comment by anasereijo** [comment]
@solazio ah ok that's fine, I will speak to them about it. thanks!

**Comment by solazio** [comment]
@caldav , @WendyS1000 can you review this please.

**Comment by SirSamTumless** [comment]
Screenshot 2019-05-30 at 17 49 42
 
The paragraph needs to be split as per the breif.

**Pull request merged**